graph_altimeter/scan/__init__.py

```diff
@@ -131,3 +138,14 @@ def generate_scan_id():
     scan_time = str(int(now.timestamp()))
     scan_id = "/".join((scan_date, scan_time, str(uuid.uuid4())))
     return scan_id
+
+
+def get_unscanned_account_error(graph_dict):
+    """Returns the error message stored in the graph dictionary generated by
+    Altimeter as the result of getting an error trying to scan one account."""
+    vertices = graph_dict["vertices"]
+    for v in vertices:
+        if v["~label"] == "error" and "error" in v:
+            error = v["error"]
+            return error
```

Comment:
```suggestion
            return v["error"]
```

Is it guaranteed that there will be only one error node? Should we look for all the error nodes and return an array?

Reply:
AFAIK. yes in the graph there should be only one error per account, as we are only scanning one account there should be only one account. So if you don't mind I would leave it as it is.

Reply:
:+1:

Reply:
Anyway assigning that variable just to return it does not add much clarity from my point of view.